src/sentry/api/exceptions.py

```diff
@@ -85,6 +85,11 @@ class SuperuserRequired(SentryAPIException):
     message = "You need to re-authenticate for superuser."
 
 
+class DataSecrecyError(SentryAPIException):
+    status_code = status.HTTP_500_INTERNAL_SERVER_ERROR
```

Comment:
@cathteng I don't think 5xx error isn't appropriate for this type of exception. 

403 Forbidden might be better: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403

Reply:
if we return 403, we will be prompting the user to login as superuser https://github.com/getsentry/sentry/blob/master/static/app/views/organizationContextContainer.tsx#L245-L255